less/availity-cards.less

```diff
@@ -63,6 +91,29 @@
   }
 }
 
+.card-header-link {
```

Comment:
What is the distiction between .card-title and .card-header?
On Sat, Sep 12, 2015 at 3:22 PM robert mcguinness <notifications@github.com>
wrote:

> In less/availity-cards.less
> <https://github.com/Availity/availity-uikit/pull/100#discussion_r39340538>
> :
>
> > @@ -63,6 +91,29 @@
> >    }
> >  }
> >
> > +.card-header-link {
>
> i'm actually thinking the name shoudl be card-link-right...it can go in a
> .card-title or card-header or card-header-secondary. thoughts?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Availity/availity-uikit/pull/100/files#r39340538>.
>